.github/workflows/pytestcov.yml

```diff
@@ -0,0 +1,30 @@
+name: Pytest with code coverage
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@master
+        with:
+          python-version: "3.8"
+
+      - name: Run image
+        uses: abatilo/actions-poetry@v2.0.0
+        with:
+          poetry-version: "1.1.11"
+
+      - name: Install python dependencies
+        run: |
+          poetry install
+      - name: Run pytest
+        run: poetry run pytest
```

Comment:
Missing code coverage

Reply:
Code coverage added temporarily. We plan to extend this workflow to push a codecoverage report to codecov.io